### PR TITLE
Hw02 express

### PR DESCRIPTION
**Pull request by wojcikowska-k**

### CODE REVIEW

File: models/contacts.js [hunk 1]
```diff
@@ -1,19 +1,81 @@
...
+    return parsedContacts;
+  } catch (error) {
+    console.log(error);
+  }
+};
 
-const addContact = async (body) => {}
+export const getContactById = async (contactId) => {
+  try {
+    const contacts = fs.readFile(contactsPath);
```
Comment (AdamChodowiecGoIt91) [inline models/contacts.js]: const contacts = `await` fs.readFile(contactsPath); can you place `await` like it is in my example and remove this one under?

File: routes/api/contacts.js [hunk 2]
```diff
@@ -1,25 +1,144 @@
...
+      });
+    } else {
+      res.json({
+        status: "error",
+        code: 404,
+        message: "Not found",
+      });
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (AdamChodowiecGoIt91) [inline routes/api/contacts.js]: what will be returned in response if this catch will be executed?

File: routes/api/contacts.js [hunk 3]
```diff
@@ -1,25 +1,144 @@
...
+    } else {
+      const contact = await addContact(responseBody.value);
+      res.status(201).json({
+        status: "success",
+        code: 201,
+        data: { contact },
+      });
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (AdamChodowiecGoIt91) [inline routes/api/contacts.js]: what will be returned in response if this catch will be executed?

File: routes/api/contacts.js [hunk 4]
```diff
@@ -1,25 +1,144 @@
...
+      });
+    } else {
+      res.json({
+        status: "error",
+        code: 404,
+        message: "Not found",
+      });
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (AdamChodowiecGoIt91) [inline routes/api/contacts.js]: what will be returned in response if this catch will be executed?

File: routes/api/contacts.js [hunk 5]
```diff
@@ -1,25 +1,144 @@
...
+      }
+    } else {
+      res.json({
+        status: "error",
+        code: 404,
+        message: "Not found",
+      });
+    }
+  } catch (error) {
+    console.log(error);
```
Comment (AdamChodowiecGoIt91) [inline routes/api/contacts.js]: what will be returned in response if this catch will be executed?

File: models/contacts.js (hunk 1 above)
Comment (wojcikowska-k) [reply models/contacts.js]: I changed position of await in every function.

File: routes/api/contacts.js (hunk 2 above)
Comment (wojcikowska-k) [reply routes/api/contacts.js]: I changed content to error.message.

File: routes/api/contacts.js (hunk 3 above)
Comment (wojcikowska-k) [reply routes/api/contacts.js]: I changed content to error.message.

File: routes/api/contacts.js (hunk 4 above)
Comment (wojcikowska-k) [reply routes/api/contacts.js]: I changed content to error.message.

File: routes/api/contacts.js (hunk 5 above)
Comment (wojcikowska-k) [reply routes/api/contacts.js]: I changed content to error.message.

File: routes/api/contacts.js [hunk 6]
```diff
@@ -58,7 +58,7 @@ router.get("/:contactId", async (req, res, next) => {
       });
     }
   } catch (error) {
-    console.log(error);
+    console.log("error", error.message);
```
Comment (AdamChodowiecGoIt91) [inline routes/api/contacts.js]: You should return something like 
```
res.json({
        status: "Internal Server Error",
        code: 500,
        message: error?.message,
      });
```

Comment (wojcikowska-k) [reply routes/api/contacts.js]: Changed code as in example

**Pull request merged**